static/usage/nav/nav-link/vue/page_three_vue.md

```diff
@@ -0,0 +1,25 @@
+```html
+<template>
+  <ion-page>
+    <ion-header>
+      <ion-toolbar>
+        <ion-buttons slot="start">
+          <ion-back-button></ion-back-button>
```

Comment:
This back button doesn't seem to be showing in the Vue Stackblitz.

Reply:
I think the problem is that the components use `<ion-page>`. That component should only be used with the router.

There's probably an improvement we can make here where developers need to reuse components in routing and non-routing contexts.

Reply:
Ah actually I think we already got an issue like this: https://github.com/ionic-team/ionic-framework/issues/23304